src/models/db/age_group.py

```diff
@@ -0,0 +1,19 @@
+from mongoengine import Document, UUIDField, \
+    ReferenceField, StringField, FloatField
```

Comment:
For multiple imports from a library/component, try to use parenthesis

```suggestion
from mongoengine import (
    Document,
    UUIDField,
    ReferenceField,
    StringField,
    FloatField
)
```